rules/falco-incubating_rules.yaml

```diff
@@ -1205,4 +1205,27 @@
     and not proc.name in (known_binaries_to_read_environment_variables_from_proc_files)
   output: Environment variables were retrieved from /proc files (file=%fd.name gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4] evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
   priority: WARNING
-  tags: [maturity_incubating, container, filesystem, process, mitre_discovery, T1083]
\ No newline at end of file
+  tags: [maturity_incubating, container, filesystem, process, mitre_discovery, T1083]
+
+- list: known_memfd_execution_binaries
+  items: []
+
+- macro: known_memfd_execution_processes
+  condition: (container_entrypoint or (proc.name in known_memfd_execution_binaries))
+
+- rule: Fileless execution via memfd_create
+  desc: >
+    Detect if a binary is executed from memory using the memfd_create technique.
+    This is a well-known defense evasion technique for executing malware on victim machine without storing payload on disk and to avoid leaving traces about what has been executed. 
+    Adopters can whitelist processes that may use fileless execution for benign purposes by adding itmes to the list `known_memfd_execution_processes`.
+    It is worth underline that this technique is used by runc to fix CVE-2019-5736. For this reason, the macro `known_memfd_execution_processes` take this into account.
+  condition: >
+    spawned_process
+    and proc.is_exe_from_memfd=true
+    and not known_memfd_execution_processes
+  output: > 
```

Comment:
```suggestion
  output: Fileless execution via memfd_create (container_start_ts=%container.start_ts proc_cwd=%proc.cwd evt_res=%evt.res proc_sname=%proc.sname gparent=%proc.aname[2] evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```

Reply:
I preserved the eligible unique fields you proposed except for the ID related ones, see Style Guide https://falco.org/docs/rules/style-guide/#output-fields.